samples/word/99-preview-apis/manage-change-tracking.yaml

```diff
@@ -0,0 +1,155 @@
+id: word-manage-change-tracking
+name: Track changes
+description: This sample shows how to get and set the ChangeTrackingMode and get reviewed text.
+host: WORD
+api_set:
+    WordApi: '1.4'
+script:
+    content: |
+        $("#get-change-tracking-mode").click(() => tryCatch(getChangeTrackingMode));
+        $("#set-change-tracking-mode").click(() => tryCatch(setChangeTrackingMode));
+        $("#get-reviewed-text").click(() => tryCatch(getReviewedText));
+        $("#setup").click(() => tryCatch(setup));
+
+        /** Get the current ChangeTrackingMode. */
+        async function getChangeTrackingMode() {
+          await Word.run(async (context) => {
+            const document = context.document;
+            document.load("changeTrackingMode");
+            await context.sync();
+
+            if (document.changeTrackingMode === Word.ChangeTrackingMode.trackMineOnly) {
+              console.log("ChangeTracking is on for just mine.");
+            } else if (document.changeTrackingMode === Word.ChangeTrackingMode.trackAll) {
+              console.log("ChangeTracking is on for everyone.");
+            } else {
+              console.log("ChangeTracking is off.");
+            }
+          });
+        }
+
+        /** Set ChangeTrackingMode. */
+        async function setChangeTrackingMode() {
+          await Word.run(async (context) => {
+            const mode = $("input[name='mode']:checked").val();
+            console.log(`Mode to set: ${mode}`);
+            if (mode === "Track only my changes") {
+              context.document.changeTrackingMode = Word.ChangeTrackingMode.trackMineOnly;
+            } else if (mode === "Track everyone's changes") {
+              context.document.changeTrackingMode = Word.ChangeTrackingMode.trackAll;
+            } else {
+              context.document.changeTrackingMode = Word.ChangeTrackingMode.off;
+            }
+
+            await context.sync();
+
+            getChangeTrackingMode();
```

Comment:
This seems redundant with the `console.log` before the `if` statement,

Reply:
Removed the other log